python/lsst/pipe/base/tests/simpleQGraph.py

```diff
@@ -166,8 +170,9 @@ def registerDatasetTypes(registry, pipeline):
         configDatasetType = DatasetType(
             taskDef.configDatasetName, {}, storageClass="Config", universe=registry.dimensions
         )
+        storageClass = "Packages" if Packages is not None else "StructuredDataDict"
```

Comment:
I had to tweak this so that the `Packages` storage class is used if the corresponding class is available. The reason is that even though I can do storage class conversions (see lsst/ctrl_mpexec#160) the `ctrl_mpexec` tests require that `butler.get()` returns `Packages` because it then uses methods to check for changes. It won't be a problem if the RFC is approved to let us migrate the `Packages` code to `utils`.